.github/workflows/auto-dependabot.yml

```diff
@@ -0,0 +1,23 @@
+name: Dependabot auto-merge
+on: pull_request_target
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Dependabot metadata
```

Comment:
What is the purpose of this step? If I'm not mistaken, it's only used to get some information from a Dependabot Pull Request, except that you never use this information through `metadata`.

Reply:
This is what gives Dependabot the lookup https://github.com/dependabot/fetch-metadata